content/terms.md

```diff
@@ -53,48 +53,50 @@ feedback](mailto:support@illumitacit.com).
+3. There is only one paid plan. The paid plan is a recurring subscription that provides access to the core features of
+   the service.
 
 4. Every generated text from a request to generate content (including, but not limited to, executing Macros, sending a
    message in a Chat based Experiment, sending a prompt in a Trial based Experiment, retrieving passages from document
-   references) will be counted towards usage on the account. This includes those originating from any request that is
+   references) will be counted towards usage on the account. Usage includes those originating from any request that is
    canceled mid process, either by using the cancel button in the UI or by closing the browser. Our underlying providers
    do not support canceling a request midway, and as such we account for words generated by the provider against the
    monthly limit on your account even though you may have canceled the request in IllumiTacit.
 
-5. You can purchase additional word capacity in your plan at any time within your account settings. Additional capacity
-   can only be purchased with the same billing cycle as your base subscription. You can not purchase words on a monthly
-   billing cycle if you are on an annual plan, and you can not purchase words on an annual billing cycle if you are on a
-   monthly plan. When adding more words into your plan, you are charged a pro rated amount between the purchase date and
-   your next billing cycle, after which you will be charged the full monthly rate. For example, if you are on an annual
-   plan that is billed on January 1st of every year and you decide to purchase more word capacity on April 1st, you will
-   pay for those words for the rest of the year at a pro rated amount (3/4 the full amount), and then will be billed
-   annually every January 1st for the full cost.
-
-6. You can lower the available word capacity in your plan at any time within your account settings. You will
-   keep the current word capacity until the end of the billing cycle, at which point the new amount will be reflected.
-   IllumiTacit does not issue refunds for changes in word capacity. Lowering the number of words that are available in
-   your plan may cause loss of capacity in your account which may make your existing usage untenable. IllumiTacit does
-   not accept any liability for such loss.
-
-7. Our payment process is conducted by our Merchant of Record ([Paddle](https://www.paddle.com/)). All fees are
+5. Usage is tracked by the number of words that are generated by our engine. Words are counted as contiguous
+   non-whitespace characters in the text. For languages that are not space-delimited, such as Chinese and Japanese,
+   words are counted as non-whitespace, non-punctuation characters in the text.
+
+6. The paid plan includes a certain number of words that are replenished into your account balance every billing cycle,
```

Comment:
I'm not sure about using the term `replenished` here. I could read it as - the number of words are updated to the max (i.e. replenished to 20k or whatever), instead of 20k words being added to whatever remaining balance you have.

However, considering the actual behavior is more generous that this possible interpretation, I don't think it's critical to reword.

Reply:
Good point. Switched to "added."